.travis.yml

```diff
@@ -1,7 +1,5 @@
 language: ruby
 cache: bundler
 rvm:
-  - 2.6.1
-  - 2.4.6
-  - 2.3.8
-  - 2.2.10
+  - 2.6
+  - 2.4
```

Comment:
For some reason, version 2.5 doesn't seem to work, at all. Not sure the correct way to resolve this but because 2.5 wasn't being targeted previously, I'm not going to consider it a blocker to suddenly add support for it. The primary goal is just to remove unsupported rubies, not add additional support.